website/src/components/views/sequencingEfforts/GenericSequencingEffortsPage.astro

```diff
@@ -91,24 +90,18 @@ const { label: subdivisionLabel, field: subdivisionField } = getLocationSubdivis
                 width='100%'
```

Comment:
We could use the new constant here. 

Also I think, this can be the small size.

Reply:
Reusing the constants will be part of my next PR anyway.

Small size is too small for some organisms (H5N1 has a lot of hosts).